### DISCUSSION TITLE
Nginx /Opera

### DISCUSSION THREAD

**Discussion by Seeker-Smith** [nginx/nginx#324] [Q&A] [answered]
I'm a newbie with Nginx so please excuse me if this if daft. Today my system started being heavily loaded. I've captured a couple screen grabs. 

What is Opera and why is it using so many resources? Thanks in advance for any insight.

The first is from webmin.
<img width="455" alt="cpu" src="https://github.com/user-attachments/assets/f94d45c7-f50c-441c-b2f0-d6c0dcead33f">
The second is from htop
<img width="1010" alt="htop" src="https://github.com/user-attachments/assets/bd1738f2-8511-4462-9cfe-8fb806759e56">

**Comment by HanadaLee** [comment]
These processes are not part of nginx, they just run as the linux user nginx. Usually it should be one of your backend web service processes.

**Comment by Seeker-Smith** [comment]
Thanks for the reply. I haven't found anything that answers why its consuming so much. Up until this afternoon the cpu and processes were a flat line.

<img width="1090" alt="cpu2" src="https://github.com/user-attachments/assets/33916737-4e06-4511-a294-c5eafa8fbb05">

**Comment by HanadaLee** [comment]
If you are not familiar with this process at all, your server may have been compromised and infected with mining software. I can google the process name and find that it may be a mining virus.

**Comment by Seeker-Smith** [comment]
Thanks
I did a search of my system for Opera. Some of the results.
<img width="863" alt="opera" src="https://github.com/user-attachments/assets/09d8db4e-5025-4b19-8e90-c29418c1c6a8">

**Reply by Seeker-Smith** [reply, marked as answer]
Fixed
